servers/rendering/renderer_rd/renderer_canvas_render_rd.cpp

```diff
@@ -1973,18 +1986,21 @@ void fragment() {
 		material_storage->material_set_shader(default_clip_children_material, default_clip_children_shader);
 	}
 
+	// cache up to 128 uniform sets
+	rid_set_to_uniform_set.set_capacity(128);
```

Comment:
Setting the cache to 128 is arbitrary. If needed, this could be a project setting, but not sure if it is needed?